src/utils/metaform-utils.ts

```diff
@@ -435,6 +435,60 @@ namespace MetaformUtils {
     return moment.duration(Math.floor(totalTime / replyCount));
   };
 
+  /**
+   * Create status section for new metaform
+   * 
+   * @returns status section for form
+   */
+  export const formStatusSection = (): MetaformField => {
+    return {
+      name: "status",
+      type: MetaformFieldType.Radio,
+      options: [
+        {
+          name: "waiting", text: ReplyStatus.WAITING, checked: true
+        },
+        { name: "processing", text: ReplyStatus.PROCESSING },
+        { name: "done", text: ReplyStatus.DONE }
+      ],
+      contexts: [
+        FormContext.MANAGEMENT_LIST
+      ],
+      flags: {
+        managementEditable: true
+      }
+    };
+  };
+
+  /**
+   * Filter out status from form
+   * 
+   * @param form Metaform form
+   * @returns Metaform without status field
+   */
+  export const removeStatusFieldFromForm = (form: Metaform) => {
+    const sectionsWithoutStatusField = form.sections?.map(section => {
+      const hasStatusField = section?.fields?.filter(field => field.name !== "status");
```

Comment:
Naming seems misleading, should it be like hasNoStatusField?

Reply:
yep